lib/components/DynamicFonts/PermissionsAcquirer.ios.ts

```diff
@@ -0,0 +1,9 @@
+export default class PermissionsAcquirer {
+  public async checkPermissions() {
+    return Promise.resolve(true);
+  }
+
+  public async getPermissions() {
```

Comment:
You not actually checking anything... Does it work on a real device?

Reply:
I only tested on a Simulator, do you expect it to work differently?

Reply:
Ok. If it works...